macros/database_specific_helpers/get_relation.sql

```diff
@@ -0,0 +1,14 @@
+{% macro get_relation(get_relation_name) %}
+    {% if execute %}
+        {% set model_get_relation_node = graph.nodes.values() | selectattr('name', 'equalto', get_relation_name) | first %}
```

Comment:
```suggestion
        {% set model_get_relation_node = graph.nodes.values() | selectattr('name', 'equalto', relation_name) | first %}
```

As per the comment above